### PR TITLE
Helps 1880 update intro whisper

### PR DESCRIPTION
**Pull request by MaggieMaSS**
Added the following components:
1. Autocomplete with single select
2. Autocomplete with multi-select
3. DateTimeInput
4. DropZone
5. Icon
6. SectionTitle
7. Breadcrumb
8. Progress
Screen Shot 2022-01-31 at 12 39 41 PM

### CODE REVIEW

File: create-loop/package.json [hunk 1]
```diff
@@ -17,9 +17,10 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@oliveai/loop-templates": "*",
+    "@oliveai/loop-templates": "file:../loop-templates",
```
Comment (JosefBud-olive) [inline create-loop/package.json:20]: Change this back to `*`

File: create-loop/package.json [hunk 2]
```diff
@@ -17,9 +17,10 @@
...
     "chalk": "^4.1.2",
     "commander": "^8.2.0",
+    "i": "^0.3.7",
```
Comment (JosefBud-olive) [inline create-loop/package.json:23]: What is this?

File: loop-templates/templates/src/whispers/IntroWhisper.test.ts.squirrelly [hunk 3]
```diff
@@ -109,12 +109,47 @@ describe('Intro Whisper', () => {
...
+      expect.objectContaining({
+        type: whisper.WhisperComponentType.Progress,
+      }),
     ];
 
     expect(actual).toEqual(expected);
   });
 
-  it('creates components with functional handlers', async () => {
+  it('creates components with functional h0andlers', async () => {
```
Comment (JosefBud-olive) [inline loop-templates/templates/src/whispers/IntroWhisper.test.ts.squirrelly:152]: I think you accidentally added a typo here in 'handlers'

File: create-loop/package-lock.json [hunk 4]
```diff
@@ -1,13 +1,1031 @@
...
-  "version": "1.1.2",
+  "version": "1.1.3",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
     "@oliveai/loop-templates": {
-      "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/@oliveai/loop-templates/-/loop-templates-1.0.0.tgz",
-      "integrity": "sha512-MwfU3oKQMi11C4yPUr++byA6bOajS/iUxfZtWygrcxd8m4cvr9y3Rx7Q3qAWsLQ6Ys+D/s1kVfnS7HfmhHck4Q=="
+      "version": "file:../loop-templates",
```
Comment (JosefBud-olive) [inline create-loop/package-lock.json:8]: Can you undo these changes to the package-lock since this still references the `file:` version :)

File: loop-templates/templates/src/whispers/IntroWhisper.ts.squirrelly [hunk 5]
```diff
@@ -170,6 +170,85 @@ export default class IntroWhisper {
...
+        const autocompleteOptions = [
+      { label: 'Value 1', value: '1' },
+      { label: 'Value 2', value: '2' },
+      { label: 'Value 3', value: '3' },
+      { label: 'Value 4', value: '4' },
+      { label: 'Value 5', value: '5' },
+    ];
+    const autocompleteSingleSelect{{it.isTypeScript ? ': whisper.Autocomplete' : ''}} = {
+      type: whisper.WhisperComponentType.Autocomplete,
+      label: 'Select 4',
```
Comment (BrettMugglinOlive) [inline loop-templates/templates/src/whispers/IntroWhisper.ts.squirrelly]: Single Select autocomplete

File: loop-templates/templates/src/whispers/IntroWhisper.ts.squirrelly [hunk 6]
```diff
@@ -170,6 +170,85 @@ export default class IntroWhisper {
...
+        console.log('Autocomplete secection changed: ', val);
+      },
+      onSelect: (error, value, onSelectWhisper) => {
+        console.log(`Received selected value: ${JSON.stringify(value)}`);
+      },
+      options: autocompleteOptions,
+    };
+    const autocompleteMultipleSelect{{it.isTypeScript ? ': whisper.Autocomplete' : ''}} = {
+      type: whisper.WhisperComponentType.Autocomplete,
+      label: 'Select 4 and 5',
```
Comment (BrettMugglinOlive) [inline loop-templates/templates/src/whispers/IntroWhisper.ts.squirrelly]: Multiselect autocomplete

**Review by BrettMugglinOlive — APPROVED** [review]

**Review by JosefBud-olive — APPROVED** [review]
